app/views/navbar.scala.html

```diff
@@ -143,7 +143,7 @@
                                 </li>
                             }
                             <li role="presentation">
-                                <a id="navbar-sign-out-btn" role="button" href='@routes.UserController.signOut("/")'>@Messages("navbar.signout")</a>
+                                <a id="navbar-sign-out-btn" role="button" href='@routes.UserController.signOut(url.getOrElse("/"))'>@Messages("navbar.signout")</a>
```

Comment:
I think that you did this so that we don't get an error when signing out from the user dashboard. But what you did here was make it so that any time you sign out, you're sent to the landing page, regardless of which page you were on when signing out. The fix I made to `UserProfileController.scala` handles this better by making sure that users are sent to the sign in page if they try to go to the user dashboard page while signed out.

Reply:
This was done in two places in this file, so the same revert is made below